scss/_root.scss

```
@@ -82,7 +82,8 @@
 }
 
 // Boosted mod
-[class*="-dark"],
+.bg-black,
+[class*="-dark"]:not(.border-dark),
 .bg-secondary {
```

```suggestion
[class*="bg-secondary"] {
```

Apparently there's no `.text-bg-black`.